tasks/tasks.go

```diff
@@ -5,13 +5,25 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"strings"
+	"sync"
+
 	"github.com/google/ts-bridge/boltdb"
 	"github.com/google/ts-bridge/datastore"
 	"github.com/google/ts-bridge/env"
 	"github.com/google/ts-bridge/stackdriver"
 	"github.com/google/ts-bridge/storage"
 	"github.com/google/ts-bridge/tsbridge"
-	"strings"
+)
+
+var (
+	sdClient     *stackdriver.Adapter
+	sdClientOnce sync.Once
+
+	statsCollector     *tsbridge.StatsCollector
+	statsCollectorOnce sync.Once
+
+	errSync []error
```

Comment:
nit: errors should be localized to the func that's returning it.

I'd suggest creating this var right before you call `sdClientOnce.Do(func()` instead of a global var.

Reply:
Agreed. Moved.